aspnet/fundamentals/servers.rst

```diff
@@ -125,42 +111,52 @@ The ``project.json`` file shown above demonstrates how to pass the ``server.urls
 
 .. code-block:: javascript
 
-	"kestrel": "Microsoft.AspNet.Hosting --server Kestrel --server.urls http://localhost:5004"
+	"web": "Microsoft.AspNet.Kestrel --server.urls http://localhost:5004"
 
 Alternately, a configuration file can be referenced, instead:
 
 .. code-block:: javascript
 
-	"kestrel": "Microsoft.AspNet.Hosting --config hosting.ini"
+	"kestrel": "Microsoft.AspNet.Hosting"
 
-Then, ``hosting.ini`` can include the settings the server will use (including the server parameter, as well):
+Then, ``hosting.json`` can include the settings the server will use (including the server parameter, as well):
 
-.. code-block:: text
+.. code-block:: json
 
-	server=Kestrel
-	server.urls=http://localhost:5004
+	{
+	  "server": "Microsoft.AspNet.Server.WebListener",
+	  "server.urls": "http://localhost:5004/"
+	}
 
 Programmatic configuration
 ^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-In addition to specifying configuration options, the server hosting the application can be referenced programmatically via the `IApplicationBuilder interface <https://github.com/aspnet/HttpAbstractions/blob/ed339a35d2a0ae6137c12e9bc8e8b037ed429bc1/src/Microsoft.AspNet.Http.Abstractions/IApplicationBuilder.cs>`_, available in the ``Configure`` method in ``Startup``. ``IApplicationBuilder`` exposes a Server property of type `ServerInformation <https://github.com/aspnet/Hosting/blob/62e534977386ebb02bfc351ec4a6bbea9ea7f635/src/Microsoft.AspNet.Hosting.Server.Abstractions/IServerInformation.cs>`_. ``IServerInformation`` only exposes a ``Name`` property, but different server implementations may expose additional functionality. For instance, WebListener exposes a ``Listener`` property that can be used to configure the server's authentication:
+In addition to specifying configuration options, the server hosting the application can be referenced programmatically via the `IApplicationBuilder interface <https://github.com/aspnet/HttpAbstractions/blob/e7bf0e71bb2ee6e08dca82d8d7485fc89e98d0b7/src/Microsoft.AspNet.Http.Abstractions/IApplicationBuilder.cs>`_, available in the ``Configure`` method in ``Startup``. ``IApplicationBuilder`` exposes Server Features of type `IFeatureCollection <https://github.com/aspnet/HttpAbstractions/blob/e7bf0e71bb2ee6e08dca82d8d7485fc89e98d0b7/src/Microsoft.AspNet.Http.Features/IFeatureCollection.cs>`_. ``IServerAddressesFeature`` only expose a ``Addresses`` property, but different server implementations may expose additional functionality. For instance, WebListener exposes ``AuthenticationManager`` that can be used to configure the server's authentication:
```

Comment:
This makes it sound like WebListener exposes AuthenticationManager off of IServerAddressesFeature.